bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/api/SecurityApi.java

```diff
@@ -91,10 +92,13 @@ public Collection<HomeEvent> getDeviceEvents(String homeId, String deviceId, Str
         throw new NetatmoException("home should not be null");
     }
 
-    public String ping(String vpnUrl) throws NetatmoException {
+    public @Nullable String ping(String vpnUrl) {
         UriBuilder uriBuilder = UriBuilder.fromUri(vpnUrl).path(PATH_COMMAND).path(SUB_PATH_PING);
-        Ping response = get(uriBuilder, Ping.class);
-        return response.getStatus();
+        try {
+            return get(uriBuilder, Ping.class).getStatus();
+        } catch (NetatmoException e) {
+            return null;
```

Comment:
Maybe a log could help.

Reply:
Would you like to add at least a DEBUG log?

Reply:
Will do, but not today, being away of my dev computer.